notifications_utils/s3.py

```diff
@@ -45,3 +45,66 @@ def s3download(bucket_name, filename):
         return key.get()["Body"]
     except botocore.exceptions.ClientError as error:
         raise S3ObjectNotFound(error.response, error.operation_name) from error
+
+
+def s3_multipart_upload_create(
+    bucket_name, file_location, tags=None, metadata=None, content_type="binary/octet-stream"
+):
+    s3 = client("s3")
+
+    args = {"Bucket": bucket_name, "Key": file_location, "ServerSideEncryption": "AES256", "ContentType": content_type}
+
+    if tags:
```

Comment:
If we have no plans to use tags or metadata, should we remove the code relating to them? It makes things more complex, and can always be added later if necessary, but there are no mentions of using these on the RFC